services/tas/tas-tests.service.js

```diff
@@ -0,0 +1,111 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+import {
+  testResultQueryParamSchema,
+  renderTestResultBadge,
+} from '../test-results.js'
+import { nonNegativeInteger } from '../../services/validators.js'
+
+const commonAttrs = {
+  namedParams: {
+    provider: 'github',
+    org: 'test-at-scale',
+    repo: 'badge-demo',
+  },
```

Comment:
Sorry I missed this before but is this a valid, public project? I'm not seeing it on GitHub and the badge is unsurprisingly showing a not found. We really like to have these examples be valid projects so that users can actually see/work with them in the modal window.

If it's something you all are planning on setting up in the near future that's alright, but otherwise it would probably be better to swap in a real target 

https://shields-staging-pr-7612.herokuapp.com/tas/tests/github/test-at-scale/badge-demo?compact_message
![](https://shields-staging-pr-7612.herokuapp.com:/tas/tests/github/test-at-scale/badge-demo?compact_message)

Using the same target as the tests would suffice too:

https://shields-staging-pr-7612.herokuapp.com/tas/tests/github/tasdemo/axios
![](https://shields-staging-pr-7612.herokuapp.com:/tas/tests/github/tasdemo/axios)

Reply:
@calebcartwright Thanks for pointing out. I've updated the example project.